src/library_glfw.js

```diff
@@ -1001,21 +1001,26 @@ var LibraryGLFW = {
         // no-op
       }
       if (i == GLFW.windows.length) {
-        var contextAttributes = {
-          antialias: (GLFW.hints[0x0002100D] > 1), // GLFW_SAMPLES
-          depth: (GLFW.hints[0x00021005] > 0),     // GLFW_DEPTH_BITS
-          stencil: (GLFW.hints[0x00021006] > 0),   // GLFW_STENCIL_BITS
-          alpha: (GLFW.hints[0x00021004] > 0)      // GLFW_ALPHA_BITS
-        }
+        if (GLFW.hints[0x00022001] > 0) { //GLFW_CLIENT_API
+          var contextAttributes = {
+            antialias: (GLFW.hints[0x0002100D] > 1), // GLFW_SAMPLES
+            depth: (GLFW.hints[0x00021005] > 0),     // GLFW_DEPTH_BITS
+            stencil: (GLFW.hints[0x00021006] > 0),   // GLFW_STENCIL_BITS
+            alpha: (GLFW.hints[0x00021004] > 0)      // GLFW_ALPHA_BITS
+          }
 #if OFFSCREEN_FRAMEBUFFER
-        // TODO: Make GLFW explicitly aware of whether it is being proxied or not, and set these to true only when proxying is being performed.
-        GL.enableOffscreenFramebufferAttributes(contextAttributes);
+          // TODO: Make GLFW explicitly aware of whether it is being proxied or not, and set these to true only when proxying is being performed.
+          GL.enableOffscreenFramebufferAttributes(contextAttributes);
 #endif
-        Module.ctx = Browser.createContext(Module['canvas'], true, true, contextAttributes);
+          Module.ctx = Browser.createContext(Module['canvas'], true, true, contextAttributes);
+        } else {
+          Module.useWebGL = false;
```

Comment:
I think this line is not needed. It will be `undefined` by default, which is falsey.